services/.eslintrc.cjs

```diff
@@ -17,5 +17,6 @@ module.exports = {
   rules: {
     'prettier/prettier': 'error',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
This might cause over-usage of any in the future. I think having this on and disabling it if we really should helps with forcing types. What do u think?

Reply:
Yeah, true. In that case I'll add exceptions for the places we already have it in